lib.php

```diff
@@ -368,6 +395,50 @@ function customcert_extend_settings_navigation(settings_navigation $settings, na
     return $customcertnode->trim_if_empty();
 }
 
+/**
+ * Build the navigation node for the 'Download certificates' item.
+ *
+ * @param boolean $isCourseNav
+ * @param \stdClass $course
+ * @param \course_context $context
+ * @return \navigation_node|null null if there are no certifcates available for the download
+ */
+function build_downloadall_node(bool $isCourseNav, stdClass $course, \context_course $context = null) {
```

Comment:
Again, this already exists in the certificate settings.

Reply:
<img width="732" height="172" alt="image" src="https://github.com/user-attachments/assets/8ab0ed7e-fac4-4875-ad3f-61bf5d944f0f" />